rllab/envs/dmcontrol_env.py

```diff
@@ -0,0 +1,130 @@
+import pygame
+import numpy as np
+from PIL import Image
```

Comment:
You don't need this dependency. You can pass the image array directly to pygame (you will need to swap axes).

Reply:
Done